src/test/java/org/wltea/analyzer/core/CharacterUtilTest.java

```diff
@@ -0,0 +1,44 @@
+package org.wltea.analyzer.core;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CharacterUtilTest {
+
+  @Test
+  public void testIdentifyCharType1() {
+    Assert.assertEquals(1, CharacterUtil.identifyCharType('1'));
+  }
+
+  @Test
+  public void testIdentifyCharType2() {
+    Assert.assertEquals(2, CharacterUtil.identifyCharType('q'));
+    Assert.assertEquals(2, CharacterUtil.identifyCharType('Q'));
+  }
+
+  @Test
+  public void testIdentifyCharType4() {
+    Assert.assertEquals(4, CharacterUtil.identifyCharType('\u4E00'));
+    Assert.assertEquals(4, CharacterUtil.identifyCharType('\uF900'));
+    Assert.assertEquals(4, CharacterUtil.identifyCharType('\u3400'));
+  }
+
+  @Test
+  public void testIdentifyCharType8() {
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\uFF01'));
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\uAC00'));
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\u1100'));
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\u3131'));
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\u3041'));
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\u30A0'));
+    Assert.assertEquals(8, CharacterUtil.identifyCharType('\u31F0'));
+  }
+
+  @Test
+  public void testRegularize() {
+    System.out.println((char) 65281);
```

Comment:
you can remove this

Reply:
whoops didn't mean to leave that in! thats why we need reviews!!